modules/openapi-generator/src/main/resources/python/__init__test.mustache

```diff
@@ -0,0 +1,13 @@
+# coding: utf-8
+
+# flake8: noqa
+{{>partial_header}}
+
+from __future__ import absolute_import
+
+__version__ = "{{packageVersion}}"
+
+# import models into model package
```

Comment:
Could we change this to:
# import TestX classes into into the test module

Reply:
Done